docs/markdown/Invoke-ScriptAnalyzer.md

```diff
@@ -399,6 +399,23 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -Fix
+Certain warnings contain a suggested fix in their DiagnosticRecord and therefore those warnings will be fixed automatically using this fix.
```

Comment:
I suggest a bit of rewording to be like other switch parameters

Fixes certain warnings which contain a fix in their DiagnosticRecord.